app/views/groups/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<p style="color: green"><%= notice %></p>
+<div class= "header">
+  <%= link_to root_path do %><%= image_tag "back.png", class:"back"%><%end%>
+  <h2>Categories</h2>
+  <%= button_to destroy_user_session_path, method: :delete, class:"off" do %> 
+    <%= image_tag "off.png", class:"off"%>
+  <%end%>
+</div>
+
+
+<div id="groups">
+  <%if @groups.any?%>
+    <% @groups.each do |group| %>
+      <%if can? :manage, group%>
+          <%= link_to group_operations_path(group), id: "group-container" do %>
+            <%= render group %>
+          <%end%>
+          <div class= "group-footers">
+            <div>
+              <small><%= group.timelapse%> ago</small>
+            </div>
+            <div class= "show-delete">
+              <%= link_to "View", group %>
+              <%= button_to "Delete", group, method: :delete, class: "delete" %>
+            </div>
+          </div>
+      <% end %>
+    <% end %>
+  <%else%>
+      <h3 style="text-align: center">Welcome! You have not added any Categories yet</h3>
+  <%end%>
+
+</div>
+
+<div class= "new-group">
+  <%= link_to "Add new category", new_group_path %>
+</div>
```

Comment:
- [ ] Please style the category or `group` index page to look like the given behance design guidelines taking note of the layout, colors, and typography as shown below 👇 
- [ ] Please ensure that the button is always at the bottom of the page and takes up the entire width of the page according to the design guideline
### Behance Design
![image](https://user-images.githubusercontent.com/46839548/227562290-ac3d8494-977d-4b8b-bcae-9e625a1494c2.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/227566885-6a683a8b-d482-4059-8109-42c2e21a93b7.png)